packages/framework/ini-files/config-specs.ini

```diff
@@ -2104,14 +2104,23 @@ use USE-PT|NO
 use USE-RDC|NO
 use USE-UVM|YES
 use USE-DEPRECATED|YES
-use PACKAGE-ENABLES|NO-EPETRA
+use PACKAGE-ENABLES|NO-PACKAGE-ENABLES
```

Comment:
Leave this one as NO-EPETRA, will prevent us from building Epetra-based packages in the meantime for PR builds.